img/s/skandia.se.svg

```diff
@@ -0,0 +1,23 @@
+     id="path2609"
+     style="fill:#007053;fill-opacity:1;fill-rule:nonzero;stroke:none" />
+  <path
+     d="M 170.46563,31.564 C 166.75313,31.564 163.76938,28.8465 163.76938,25.52025 C 163.76938,22.1965 166.75313,19.47525 170.46563,19.47525 C 174.18438,19.47525 177.16688,22.1965 177.16688,25.52025 C 177.16688,28.8465 174.18438,31.564 170.46563,31.564 M 170.46563,16.32275 C 166.75313,16.32275 163.76938,13.6015 163.76938,10.27775 C 163.76938,6.954 166.75313,4.23275 170.46563,4.23275 C 174.18438,4.23275 177.16688,6.954 177.16688,10.27775 C 177.16688,13.6015 174.18438,16.32275 170.46563,16.32275"
+     id="path2613"
+     style="fill:#aba38f;fill-opacity:1;fill-rule:nonzero;stroke:none" />
+</svg>
+<!-- version: 20090314, original size: 177.16689 31.563999, border: 3% -->
```

Comment:
I recommend you use their favicon instead as it will be more recognizable at the resolution we use.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" style="fill:#fff;background:#007053" viewBox="0 0 45 45"><ellipse id="a" cx="22.5" cy="14.8" rx="6.7" ry="6"/><use y="15.3" href="#a"/></svg>
```